training/src/main/java/wolox/training/exceptions/BookNotFoundException.java

```diff
@@ -0,0 +1,20 @@
+package wolox.training.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+@ResponseStatus(code = HttpStatus.NOT_FOUND, reason = "Book Not Found")
+public class BookNotFoundException extends RuntimeException {
+    public BookNotFoundException() {
+        super();
+    }
+    public BookNotFoundException(String message, Throwable cause) {
+        super(message, cause);
+    }
+    public BookNotFoundException(String message) {
+        super(message);
+    }
+    public BookNotFoundException(Throwable cause) {
+        super(cause);
```

Comment:
You can declare the message in the constructor
```suggestion
        super( "Book not found");
    }
```